api/src/legacy_soap_api/legacy_soap_api_client.py

```diff
@@ -232,6 +235,18 @@ def GetApplicationZipRequest(self) -> grantors_schemas.GetApplicationZipResponse
             soap_config=self.operation_config,
         )
 
+    def GetSubmissionListExpandedRequest(
```

Comment:
Functions shouldn't be capitalized, this would be misinterpreted as a class.

```suggestion
    def get_submission_list_expanded(
```

Why does this optionally return a list?

Reply:
This was a consequence of code calling the operation_name 
```
        operation_method = getattr(self, self.operation_config.request_operation_name))
```
Where the operation_name is camel case like `GetApplicationZipRequest`. I've updated the methods to snake case.